.github/actions/purge-m365-user-data/action.yml

```diff
@@ -56,4 +56,6 @@ runs:
         ./foldersAndItems.ps1 -WellKnownRoot root -User ${{ inputs.user }} -FolderNamePurge PersonMetadata 
         ./foldersAndItems.ps1 -WellKnownRoot deleteditems -User ${{ inputs.user }} 
         ./foldersAndItems.ps1 -WellKnownRoot recoverableitemsroot -User ${{ inputs.user }} -FolderNamePurge Audits 
-        ./foldersAndItems.ps1 -WellKnownRoot recoverableitemsroot -User ${{ inputs.user }} -FolderNamePurge Purges
\ No newline at end of file
+        ./foldersAndItems.ps1 -WellKnownRoot recoverableitemsroot -User ${{ inputs.user }} -FolderNamePurge Purges
+        ./foldersAndItems.ps1 -WellKnownRoot recoverableitemsroot -User ${{ inputs.user }} -FolderNamePurge Deletions
+        ./foldersAndItems.ps1 -WellKnownRoot recoverableitemsroot -User ${{ inputs.user }} -FolderNamePurge "Calendar Logging"
```

Comment:
`Deletions` is a good addition. This is the equivalent of emptying the Recoverable items folder from the UI. 

`Calendar Logging` has less clear consequences in terms of impact on functionality - https://learn.microsoft.com/en-us/exchange/policy-and-compliance/recoverable-items-folder/recoverable-items-folder?view=exchserver-2019

Reply:
Maybe pull out Calendar Logging and see what the impact is first.

Reply:
done